meta/documents/changelog_en.md

```diff
@@ -1,5 +1,10 @@
 # Release Notes for Elastic Export BeezUp
 
+## v1.0.15 (2017-09-18)
+
+### Fixed
+- An issue was fixed which caused multiple header rows were exported. 
```

Comment:
besser wäre evtl "An issue was fixed which caused multiple header rows to be exported"